python/shotgun_model/shotgun_standard_item.py

```diff
@@ -8,16 +8,15 @@
 # agreement to the Shotgun Pipeline Toolkit Source Code License. All rights
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
-from .util import sanitize_qt, get_sg_data
 
 from tank.platform.qt import QtGui
 
+from .util import sanitize_qt
+from .util import get_sg_data as util_get_sg_data
 
 class ShotgunStandardItem(QtGui.QStandardItem):
     """
     Special implementation of StandardItem which bridges PyQt and PySide.
-
-    Do not construct this object directly - instead use the ShotgunModel.create_item() method.
```

Comment:
So there wasn't actually a `create_item()` method on `ShotgunModel`. The `__create_item()` method exists, but is private. Maybe that's why I removed it? 

You're right though, there should be some comment. I've added a warning here and in `ShotgunHierarchyItem`. Maybe that's sufficient?